components/server/src/user/user-deletion-service.ts

```diff
@@ -148,6 +149,9 @@ export class UserDeletionService {
         user.avatarUrl = "deleted-avatarUrl";
         user.fullName = "deleted-fullName";
         user.name = "deleted-Name";
+        if (user.verificationPhoneNumber) {
+            user.verificationPhoneNumber = "deleted-verificationPhoneNumber";
```

Comment:
`ER_DATA_TOO_LONG: Data too long for column 'verificationPhoneNumber'`

we're almost there, just 2 chars too long.

Reply:
what did you type in?

Reply:
ah, got it.

Reply:
That happens on deletion of an account, which actually breaks the process.

Reply:
`deleted-phoneNumber` with a length of 20 should work just fine.